extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -131,6 +131,10 @@ static public function newFromIds( Array $ids ) {
 		return array_values( $wallMessages );
 	}
 
+	static private function isWallMessage( $title ) {
+		return in_array( $title->getNamespace(), [ NS_USER_WALL_MESSAGE, NS_USER_WALL_MESSAGE_GREETING ] );
```

Comment:
From what I have seen forum checks namespace, but if you know place where it should also be applied please give me a link ;)